packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/DeviceHeaderButton/index.tsx

```diff
@@ -1,82 +1,53 @@
-import { Translation } from '@suite-components';
+import { colors, Icon } from '@trezor/components';
+import { NotificationCard, Translation } from '@suite-components';
 import * as deviceUtils from '@suite-utils/device';
 import { TrezorDevice } from '@suite-types';
 
-const Row = styled.div`
-    display: flex;
-    align-items: center;
+const GrayNotificationCard = styled(props => <NotificationCard {...props} />)`
+    background: ${colors.NEUE_BG_GRAY};
 `;
-
-const Attention = styled.div`
-    display: flex;
-    font-size: ${variables.FONT_SIZE.TINY};
-    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
-    color: ${colors.RED_ERROR};
-    margin-right: 20px;
-`;
-
-const AttentionIconWrapper = styled.div`
-    margin-right: 1ch;
-`;
-
 interface Props {
+    needsAttention: boolean;
     device: TrezorDevice;
     deviceStatusMessage?: React.ReactNode;
     onSolveIssueClick: () => void;
     onDeviceSettingsClick: () => void;
 }
 
-const SolveIssueButton = ({ onSolveIssueClick, deviceStatusMessage, device }: Props) => (
-    <Row>
-        <Attention>
-            <AttentionIconWrapper>
-                <Icon icon="WARNING" size={16} color={colors.RED_ERROR} />
-            </AttentionIconWrapper>
-            {deviceStatusMessage && <>{deviceStatusMessage}</>}
-        </Attention>
-        <Button
-            variant="secondary"
-            onClick={onSolveIssueClick}
-            data-test={`@switch-device/${device.path}/solve-issue-button`}
-        >
-            <Translation id="TR_SOLVE_ISSUE" />
-        </Button>
-    </Row>
-);
-
-const DeviceSettingsButton = ({ onDeviceSettingsClick }: Props) => (
-    <Icon
-        usePointerCursor
-        size={24}
-        icon="SETTINGS"
-        color={colors.NEUE_TYPE_LIGHT_GREY}
-        onClick={onDeviceSettingsClick}
-    />
-);
-
 const DeviceHeaderButton = (props: Props) => {
     const { device } = props;
     const deviceStatus = deviceUtils.getStatus(device);
     const deviceStatusMessage = deviceUtils.getDeviceNeedsAttentionMessage(deviceStatus);
     const isUnknown = device.type !== 'acquired';
-    const needsAttention = deviceUtils.deviceNeedsAttention(deviceStatus);
-
-    if (needsAttention)
-        return (
-            <SolveIssueButton
-                deviceStatusMessage={
-                    deviceStatusMessage ? <Translation {...deviceStatusMessage} /> : undefined
-                }
-                {...props}
-            />
-        );
-
-    if (!isUnknown) return <DeviceSettingsButton {...props} />;
 
-    return null;
+    return (
+        <>
+            {props.needsAttention && (
+                <GrayNotificationCard
+                    variant="warning"
+                    button={{
+                        children: <Translation id="TR_SOLVE_ISSUE" />,
+                        onClick: props.onSolveIssueClick,
+                        dataTest: `@switch-device/${device.path}/solve-issue-button`,
```

Comment:
wouldn't be better to use `data-test` instead of introducing `dataTest` prop in `Button` component?